### ISSUE TITLE
Armorpaint doesn't save my work

### ISSUE THREAD

**Issue by meumeu66** [armory3d/armorpaint#368]
*ArmorPaint 0.8-0fcfa61, Windows*

Hello, 
Sorry, I'm noob with armorpaint, but I purchased and installed Armorpaint a few weeks ago and couldn't make it work.
Each time I try to save my work, everything looks okay, it creates an .arm file quite heavy, which seems okay if it saves textures inside it, but whenever I try to reopen it I only get the white cube, evrything is gone: textures, created materials etc...
I am doing something wrong? It's really deaspointing so far and I didn't find such similar problems while looking on the web, so thanks for any help!

**Comment by luboslenco** [comment]
Hi, thanks for report. What happens if you check out the [samples repository](https://github.com/armory3d/armorpaint_samples) ([download link](https://github.com/armory3d/armorpaint_samples/archive/master.zip)), and open the `sample.arm` project which it contains? Does it work or do you still get white cube?

**Comment by meumeu66** [comment]
Hi,
Thanks for your fast reply!
If I double click the file "sample.arm" nothing happens, I still get the
white cube.
But by launching armorpaint before, and opening the file via file->Open, it
manages to open it correctly. But by doing exactly the same with my
personal projects, they still don't open: only the white cube and no
materials.
I precise I try on a pretty decent computer with Windows 10 Pro: 2x 1080
Ti, AMD ryzen 7 1800X and 32 Go RAM). I try to use the exe file, I don't
know how to build it myself.
Any idea?

Thanks!


Le jeu. 16 avr. 2020 à 10:13, Lubos Lenco <notifications@github.com> a
écrit :

**Comment by luboslenco** [comment]
- Are you able to save a simple `.arm` project on your machine and share it with me? After putting it into `.zip` you should be able to attach it directly here on GitHub (but it does not work through email attachments).
- Can you spot a `stderr.txt` file generated in the folder where your `ArmorPaint.exe` is located?

Many thanks for additional info!

**Comment by meumeu66** [comment]
Hello!
I really appreciate your help, many thanks!
Here is a small project I just made with only 1 imported mesh and 1 texture, which doesn't reopen after saving and closing it.
I've put 2 "stderr.txt" into the .zip file: one from the exe folder, the other from the project location.

[AP_TestProject.zip](https://github.com/armory3d/armorpaint/files/4491923/AP_TestProject.zip)

**Comment by luboslenco** [comment]
Hi, have you had a chance to test again with some of the more recent builds? Does the issue persist? If so, is there any chance that some anti-virus software could interfere with ArmorPaint?

**Comment by meumeu66** [comment]
Hello,

I'm sorry but I'm pretty sure I made an answer to that thread, once I got it morking, to prevent other users having the same issue, but it seems it didn't get saved too :')
It's been a while, but as far as I remember, il was due to the folder names or/and most likely to the textures filenames I used. I had tagged some info in my texture library, often between brackets like resolution, for example: [4K]TextureNameAlbedo.jpg etc., and it seems armorpaint doesn't recognize such characters. By renaming both my Filenames and directory path I finally got it working.

Thanks for you help, it might be something to be improved in further releases (I didn't try the latest though, maybe it's already done...) and I hope this thread could help others with similar issues.
